Sources/PointFree/Emails/SharedEmailComponents.swift

```diff
@@ -31,13 +32,41 @@ let emailFooterView = View<Prelude.Unit> { _ in
 
           p([`class`([Class.pf.type.body.small])], [
             "Our postal address: 139 Skillman #5C, Brooklyn, NY 11211"
-            ])
-          ])
+            ]),
+
+          ] + unsubscribeView.view((user, newsletter)))
```

Comment:
unsubscribe info is contained in another view to make it easy to omit its contents when we dont need an unsubscribe link, i.e. all of our account/payment related emails.